admin/config-ui/fields/class-field-choice.php

```diff
@@ -24,17 +24,17 @@ public function __construct( $field ) {
 	/**
 	 * Add a choice to the properties
 	 *
-	 * @param string $value              Value op the option.
-	 * @param string $label              Label to display for the value.
-	 * @param string $screen_reader_text Optional. Screenreader text to use.
+	 * @param string $value          Value op the option.
+	 * @param string $label          Label to display for the value.
+	 * @param string $friendly_label Optional. Human friendly label text to use.
```

Comment:
I get that the text is not for "screenreaders", but this isn't quite the description that feels right.
Suggested "Explanatory label".